docs/registry/create/_index.md

```diff
@@ -834,6 +1109,25 @@ fn (c *component) someFunction(a int) {
 }
 ```
 
+{{% /tab %}}
+{{% tab name="C++" %}}
+
+To enable your C++ module to write log messages to the Viam app, add the following lines to your code:
+
+```cpp {class="line-numbers linkable-line-numbers"}
+// Include the boost trivial logger:
+#include <boost/log/trivial.hpp>
+
+// Within your main() program, before logging anything, accept command line arguments to control log severity:
+set_logger_severity_from_args(argc, argv);
```

Comment:
Q: Or is this only required when performing `debug`-level logging, and not required otherwise?